test/framework/util.go

```diff
@@ -39,7 +40,7 @@ const (
 // GenerateNamespaceName generates a namespace name, taking configuration into account (local or not)
 func GenerateNamespaceName(prefix string) string {
 	rand.Seed(time.Now().UnixNano())
-	ns := fmt.Sprintf("%s-%s", prefix, RandSeq(4))
+	ns := fmt.Sprintf("%s-%s", prefix, test.GenerateShortUID(4))
```

Comment:
This change breaks the Cucumber tests when running in parallel.
In case of parallel tests there is a chance that tests will start in exactly same time (happens in the beginning of test execution). In such case same id is provided resulting in same namespace name, leading to undesired behaviour.

Can you please return RandSeq back for Cucumber tests?

Reply:
Of course! But I'm wondering why the UID is generated with the same characters? Could you please explain a little bit more (I'll ping you)?